src/backend/distributed/sql/citus--9.3-2--9.4-1.sql

```diff
@@ -0,0 +1,3 @@
+/* citus--9.3-2--9.4-1 */
+
+/* bump version to 9.4-1 */
```

Comment:
```suggestion
-- citus--9.3-2--9.4-1

-- bump version to 9.4-1
```

Reply:
Thanks for doing this.

I plan to create a style-check script that disallows migration files composing of only whitespaces and C-style comments.